submissions/xT0PERx/Popup-style/index.html

```diff
@@ -4,130 +4,134 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>Задание на tullbar</title>
-
+        <title>Popup-style xT0PERx</title>
         <link rel="stylesheet" type="text/css" href="style.css" />
     </head>
     <body>
-        <div class="container">
-            <div class="div1"><p>Mail</p></div>
-            <div class="div2"><p>Images</p></div>
-            <div class="main_tullbar">
-                <label for="first"
-                    ><img
-                        src="https://img.icons8.com/nolan/64/pincode-keyboard.png"
-                /></label>
-                <input id="first" type="checkbox" />
-                <div class="icons">
-                    <div tabindex="0">
-                        <img src="images\apps.png" />
-                        <p>Apps</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\blogs.png" alt="" />
-                        <p>Blogs</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\calendar.png" />
-                        <p>Calendar</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\chat.png" alt="" />
-                        <p>Chat</p>
-                    </div>
-
-                    <div tabindex="0">
-                        <img src="images\classroom.png" />
-                        <p>Classroom</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\contacts.png" alt="" />
-                        <p>Contacts</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\documents.png" />
-                        <p>Documents</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\files.png" alt="" />
-                        <p>Files</p>
-                    </div>
-
-                    <div tabindex="0">
-                        <img src="images\mail.png" />
-                        <p>Mail</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\maps.png" alt="" />
-                        <p>Maps</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\mic.png" />
-                        <p>Microphone</p>
-                    </div>
-                    <div tabindex="0">
-                        <img src="images\news.png" alt="" />
-                        <p>News</p>
-                    </div>
-
-                    <input id="second" type="checkbox" />
-                    <label for="second">More</label>
+        <nav class="container">
+            <ol>
+                <li>
+                    <a class="mail" href="#" tabindex="1"><p>Mail</p></a>
```

Comment:
All interactive elements already have a focus behavior by default. Reasonable reason to use attribute `tabindex`  on them is redefine default order of focused elements.

Reply:
Does this mean that I do not use a tab index for them in this case?

Reply:
Yes, by default `a` tag has a focus in the browser and you can iterate them without `tabindex`.
[Video](https://share.vidyard.com/watch/bMw9c5SdBubQGKWzLwCTY2?)